examples/seismic/skew_self_adjoint/utils.py

```diff
@@ -1,82 +1,159 @@
-from timeit import default_timer as timer
+from sympy import exp, Min
 import numpy as np
-from devito.builtins import gaussian_smooth
+from devito import (Grid, Constant, Function, SpaceDimension, Eq, Operator)
+from examples.seismic import TimeAxis
+# from devito.builtins import gaussian_smooth
 
+__all__ = ['critical_dt', 'setup_wOverQ', 'defaultSetupIso']
 
-def setup_wOverQ(wOverQ, w, qmin, qmax, npad, sigma=None):
+
+def critical_dt(v):
```

Comment:
`compute_critical_dt`?

otherwise you have:

* the function -- called `critical_dt`
* the variable -- called `dt_critical` (actually, currently it's `dtCritical` or `criticalDt`, but as I said it should change for homogeneity)

Reply:
changed name, although will go away when I refactor to use Model and AcquisitionGeom